nengo/params.py

```diff
@@ -416,7 +417,7 @@ def validate(self, instance, function_info):
         super(FunctionParam, self).validate(instance, function)
 
 
-class FrozenObject(object):
+class FrozenObject(with_metaclass(DocstringInheritor)):
```

Comment:
Hm... are we sure we actually want this on the `FrozenObject` base class? I think it could be confusing if a user subclasses dist, for example, and doesn't write a docstring... calling `help` will give something wrong instead of blank.

The alternative is to do multiple inheritance in the solver; i.e., `class Solver(FrozenObject, with_metaclass(DocstringInheritor)):`.